### PR TITLE
Update infinite scrolling

### PR DESCRIPTION
**Pull request by kienan**
**What & Why**:
Update infinite scrolling following the new scrolling style changes. I added a little loading wheel animation as well to create a fake loading period that makes infinity bearable.

I converted from tabs to spaces so the whole file is overwritten. The lines I actually added are 102-110. Also, this file needs a hardcore refactor later on -- style is all over the place, etc.

### CODE REVIEW

**Comment by atallahhezbor** [comment]
Do we have any set style guide yet? Perhaps we should hold off on refactoring style until we settle that.

The fix itself looks good though. Tested and works

File: app/assets/javascripts/courses.js [hunk 1]
```diff
@@ -2,367 +2,370 @@ var ready;
...
+    return
+  }
+
+  window.onscroll = function(ev) {
+    if ((window.innerHeight + window.scrollY) >= document.body.offsetHeight && $('.reviews-box').children('.hidden').length > 0) {
+      $('.wheel-loader').removeClass('hidden');
+      setTimeout(function () {
+        appendReviews();
+        $('.wheel-loader').addClass('hidden');
+      }, 800);
```
Comment (atallahhezbor) [inline app/assets/javascripts/courses.js]: Not sure if this delay is better or worse for usability. To me it felt more laggy and jumpy
@aw3as @emumoo @DC00 @Apoorva-A Thoughts?

**Comment by kienan** [comment]
Yeah agreed about holding off on refactoring.

File: app/assets/javascripts/courses.js (hunk 1 above)
Comment (kienan) [inline app/assets/javascripts/courses.js]: I based the slight delay on this [article](https://www.smashingmagazine.com/2016/03/pagination-infinite-scrolling-load-more-buttons/?utm_source=html5weekly&utm_medium=email). Pretty much that immediate, undelayed infinite scrolling isn't the best approach. Really, it says to add a show more button, but I think the slight lag is a good middle ground?

**Comment by kienan** [comment]
Note: don't merge until I fix the sidebar on small screens.

File: app/assets/javascripts/courses.js (hunk 1 above)
Comment (Apoorva-A) [inline app/assets/javascripts/courses.js]: Just saw this.. doesn't seem too bad

**Comment by DC00** [comment]
Infinite scrolling works for me. Delay feels fine

Side note: made me think we should display the # of comments on the Professor cards

**Comment by kienan** [comment]
@atallahhezbor @emumoo @DC00 @aw3as this should fix infinite scrolling and also the sidebar issues. let me know how it looks so we can get this merged.

**Comment by emumoo** [comment]
The sidebar looks good but the alpha and beta tags are unaligned.

sidebar

**Comment by emumoo** [comment]
Let's merge this, I can't.

**Comment by kienan** [comment]
@emumoo i'll look at fixing the tags today and merge it hopefully

**Pull request merged**